automatic/kubescape/update.ps1

```diff
@@ -0,0 +1,34 @@
+[CmdletBinding()]
+param([switch] $Force)
+
+Import-Module AU
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
+}
+
+function global:au_SearchReplace {
+  @{
+    ".\legal\VERIFICATION.txt" = @{
+      "(?i)(^\s*64\-bit software.*)\<.*\>" = "`${1}<$($Latest.URL64)>"
+      "(?i)(^\s*checksum\s*type\:).*"      = "`${1} $($Latest.ChecksumType64)"
+      "(?i)(^\s*checksum(64)?\:).*"        = "`${1} $($Latest.Checksum64)"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $LatestRelease = Get-GitHubRelease kubescape kubescape
+
+  $checksumAsset = $LatestRelease.assets | Where-Object {$_.name -eq 'kubescape-windows-latest.sha256'} | Select-Object -ExpandProperty browser_download_url
+  $checksum64 = Invoke-WebRequest -Uri $checksumAsset -UseBasicParsing
```

Comment:
Checksum should not be calculated in the `global:au_GetLatest` section.

Instead, the checksum calculation should be done in `global:au_BeforeUpdate` which the call to `Get-RemoteFiles` already does.